nvtabular/torch_dataloader.py

```diff
@@ -272,9 +363,11 @@ def __init__(self, paths, **kwargs):
         self.cur_path = None
         self.kwargs = kwargs
         self.rows = 0
+        batch_size = kwargs.get("sub_batch_size", 1)
         for file_path in self.paths:
             (num_rows, num_row_groups, columns,) = cudf.io.read_parquet_metadata(file_path)
-            self.rows += (num_rows // kwargs.get("sub_batch_size", 1)) + 1
+            chunks = int(num_rows / batch_size)
+            self.rows += chunks + 1 if num_rows % batch_size > 0 else chunks
```

Comment:
The chance of this actually causing problems is small, but it's worth mentioning that the (int to float to int) could possibly cause problems. If we're okay with this, it'd probably be worth replacing with `chunks = math.ceil(num_rows / batch_size)`. But I'd instead suggest something like

```
chunks, spill = divmod(num_rows, batch_size)
if spill > 0: chunks += 1
self.rows += chunks
```

Also, I'm finding the names here confusing. Is it worth clarifying a bit? E.g. does a "row" correspond to a batch? And does the word "chunk" here means something completely different than the "chunk" in the `ChunkQueue`?